workshop1/01-setup.md

```diff
@@ -4,60 +4,127 @@
+[React] is a JavaScript library for building interactive user interfaces for
+the web. We're going to use it to build our animal shelter front-end.
 
-With your repo name in hands (it will be your working repo for the duration of the project), let's get started!
+React provides a tool called [`create-react-app`] which generates the code
+needed for a basic React application with a single command. We are going to
+clone (or "check out") our repository so that we have a local copy of it, and
+then use [`create-react-app`] to generate the scaffolding for our app.
 
- 1. Create a local folder to keep your code. Ex: C:\CodessInTheClassroom.
+### Cloning your repository
 
- 2. Find your repo at [CodessInTheClassrom GitHub project](https://github.com/codessintheclassroom).
+You should have received a slip of paper with the name of your personal
+repository on it. The name will be something like `happy-rover`, and you'll
+be able to find it within the [CodessInTheClassroom GitHub organisation].
 
- 3. Clone your repository: instructions available [here](../git-cheatsheet.md#clone).
+To clone your repository, open up PowerShell (or your preferred shell):
 
-At this stage you have your code checked out locally, and by default you would be working off the `master` branch. We should avoid committing and pushing code straight to `master`, as this branch is usually where your stable code should be (we normally use `master` as a source for production code). We'll talk about this in a bit more detail when we discuss Pull Requests.
-Having that in mind, we will now:
+1. Create a local folder to keep your code and change to it. For example, to
+   create `C:\CodessInTheClassroom`:
 
- 1. Create a new branch for the next coding steps.
-   * Make sure you have a descriptive branch name. Ex: `your-username/workshop1-adding-react-app`.
-   * Check the [git cheatsheet](../git-cheatsheet.md#create-a-branch) for instructions.
+       mkdir C:\CodessInTheClassroom
+       cd C:\CodessInTheClassroom
 
- 2. Install the latest Long Term Support (LTS) version of node js from https://nodejs.org/en/. Currently this is version 10.
-   * Avoid non LTS versions, e.g. version 11, since they tend to have bugs.
+2. Find your repository within the [CodessInTheClassroom GitHub organisation].
 
-### Starting the app
- This section will give you the steps to generate the base code to start our application.
- We'll start by creating a [React](https://reactjs.org/) app:
+3. Clone your repository using the instructions in [our Git
+   cheatsheet](../git-cheatsheet.md#clone). For example:
 
-1. Launch Visual Studio Code and use `File -> Open Folder` and open the folder `front-end`. It will be under the folder we created on the previous sections (Ex: CodessInTheClassroom).
+       git clone https://github.com/codessintheclassroom/happy-rover.git
 
-2. Open a terminal by clicking on `Terminal -> New Terminal`.
+### Creating the application scaffolding
```

Comment:
Another minor thing: we're only defining scaffolding below, so perhaps we can move that definition to here? Or perhaps just a brief explanation?

(I'm only saying this because I myself was unaware of this term, and a google translate told me this was a scaffolding in civil construction term :P )

Reply:
Yup, will do!